compiler/record-minmax/README.md

```diff
@@ -0,0 +1,18 @@
+# record-minmax
+
+_record-minmax_ is a tool to embed min/max values of activations to the circle model for post-training quantization.
+
+## Usage
+
+This will run with the path to the input model (.circle), input data (.h5), and the output model (.circle).
```

Comment:
What about passing a bunch of `input data`? Feeding just one image is quite an expensive operation, as you will need the following operations to be performed for each input:

1. load the model from file;
2. construct the interpreter;
3. call the interpreter;
4. store the model to file.

On the other hand, if passing a bunch of `input data`, we will only need to do every operation just once, except for (3), which will be repeated N times.

Reply:
You're right. And that's just what I want to do. :) The input data (.h5 file) will contain multiple (~1000) images, so the step 3 will be repeated in the loop.

The term 'input data' may have confused you. I will update the README.

Reply:
Oh, I see)
In that case, please check out my other comment.